pom.xml

```diff
@@ -210,6 +215,11 @@
       <version>1.8.4</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+      <version>4.5.2</version>
+    </dependency>
```

Comment:
The only reference I can find to this dependency doesn't appear to be used. Can this be removed?

Reply:
It is used under the hood by WebDriver. If I don't include it, I get a runtime error:
java.lang.NoClassDefFoundError: org/apache/http/cookie/CookieSpecProvider